openmc/mgxs/mgxs.py

```diff
@@ -1998,7 +1998,7 @@ def tallies(self):
         """
 
         # Instantiate tallies if they do not exist
-        if super(NuScatterMatrixXS, self).tallies is None:
+        if super(ScatterMatrixXS, self).tallies is None:
```

Comment:
I think you're right on the transparency and readability with `self._tallies`. And yes, `create_tallies` is now buried inside all of the `tallies` property getters so it should be private. I've pushed both changes and think this PR is ready to go.